pipeline.py

```diff
@@ -51,19 +58,27 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self._storage = {}
+        self.path = Path(path_to_raw_txt_data)
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path_to_raw = self.path
+
+        for file in path_to_raw.glob('*'):
+            if 'raw.txt' in file.name:
```

Comment:
instead of filtering like that, you can iterate over relevant files only by specifying filename pattern as an argument passed to glob

Reply:
fixed, hope I understood you correctly